tests/Oryx.Tests.Common/DockerVolume.cs

```diff
@@ -53,24 +53,21 @@ private DockerVolume(string originalHostDir, string mountedHostDir, string conta
         /// </summary>
         /// <param name="hostDir">local directory to be used in a container</param>
         /// <returns>DockerVolume instance that can be used to mount the new copy of `originalDir`.</returns>
-        public static DockerVolume CreateMirror(string hostDir)
+        /// <param name="writeToHostDir">a boolean which indicates if we want the actual directory or the copy of the actual directory</param>
+        public static DockerVolume CreateMirror(string hostDir, bool writeToHostDir = false)
         {
             if (string.IsNullOrEmpty(hostDir))
             {
                 throw new ArgumentException($"'{nameof(hostDir)}' cannot be null or empty.");
             }
-
```

Comment:
A lot of empty lines were removed in this file -- would you mind adding them back to make the file easier to read?